src/commands/config/wallets/add.ts

```diff
@@ -57,17 +54,17 @@ export default class ConfigWalletAddsCommand extends Command {
     } else {
       const existingName = config.wallets.find((wallet) => wallet.name === args.name)
       if (existingName) {
-        this.error(`Wallet with name ${args.name} already exists`)
+        this.error(`A wallet named '${args.name}' already exists.\nUse a different name or run 'mplx config wallets remove ${args.name}' to remove the existing wallet first.`)
       }
 
       const existingPath = config.wallets.find((wallet) => wallet.path === args.path)
       if (existingPath) {
-        this.error(`Wallet with path ${args.path} already exists`)
+        this.error(`This wallet file is already configured as '${existingPath.name}'.\nUse 'mplx config wallets set ${existingPath.name}' to switch to it.`)
       }
 
       const existingAddress = config.wallets.find((wallet) => wallet.address === signer.publicKey.toString())
       if (existingAddress) {
-        this.error(`Wallet with address ${shortenAddress(signer.publicKey)} already exists`)
+        this.error(`This wallet address (${shortenAddress(signer.publicKey)}) is already configured as '${existingAddress.name}'.\nUse 'mplx config wallets set ${existingAddress.name}' to switch to it.`)
```

Comment:
```suggestion
        this.error(`The wallet address (${shortenAddress(signer.publicKey)}) is already configured as '${existingAddress.name}'.\nUse 'mplx config wallets set ${existingAddress.name}' to switch to it.`)
```
Wouldn't this be "better" english?